code/modules/clothing/spacesuits/void/excelsior.dm

```diff
@@ -22,19 +18,16 @@
 /obj/item/clothing/suit/space/void/excelsior
 
 	name = "Excelsior armor"
-	desc = "An ancient space suit design, reworked to include significant protection, especially against energy discharges"
+	desc = "An ancient space suit design, remade with advanced materials. Provides good protection, especially against energy discharges"
 	icon_state = "soviet_skaf"
 	item_state = "soviet_skaf"
-	slowdown = 1.5
+	slowdown = 0.8
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
 	flags_inv = HIDEGLOVES|HIDEJUMPSUIT|HIDETAIL
 	w_class = ITEM_SIZE_NORMAL
 	//Decent all around, but less ballistic resistance
-	armor = list(melee = 60, bullet = 50, laser = 70, energy = 75, bomb = 55, bio = 100, rad = 90)
+	armor = list(melee = 45, bullet = 45, laser = 55, energy = 55, bomb = 25, bio = 100, rad = 90)
 	siemens_coefficient = 1 //Shockproof!
```

Comment:
The code comment states that it is "Shockproof!", but the `siemens_coefficient` equals `1`. To be shockproof it must be equal to zero, this line currently makes it more at suspect to tasers than a regular space suit. I recommend lowering it to a value of .7 or .6, or having the line removed. Same as the helmet in line 17.

Reply:
thanks for that, i wasnt clear on how it worked. But yes i do want them to be entirely shockproof, excelsior will work a lot with machinery and we want them generally dying in combat, not electrical accidents